src/potdata/transformations.py

```diff
@@ -335,6 +336,61 @@ def run_md(
 
         return structures
 
+class ACEMDTransformation(BaseMDTransformation):
+    """Molecular dynamics transformation using m3gnet and ACE.
+
+    Args:
+        taut (float): time constant for Berendsen temperature coupling
+        loginterval (int): write to log file every interval steps
+        append_trajectory (bool): Whether to append to prev trajectory
+
+    """
+
+    @requires(
+        pyace,
+        "`ACE` is needed for this transformation. To install it, see "
+        "https://pacemaker.readthedocs.io/en/latest/",
+    )
+    def run_md(
+        self,
+        structure: Structure,
+        #potential: [Union[Potential, str]],
+        trajectory_filename: str = "md.traj",
+        log_filename: str = "md.log",
+        timestep: float = 1.0,
+        taut: Optional[float] = None,
+        loginterval: int = 1,
+        append_trajectory: bool = False,
+    ) -> list[Structure]:
+        from ase.io import Trajectory as Trajectory
+        from ase.md.nvtberendsen import NVTBerendsen
+        from ase import units
+        from pymatgen.io.ase import AseAtomsAdaptor
+        from pyace import PyACECalculator
+
+        atoms = AseAtomsAdaptor.get_atoms(structure)
+        # Initialize ACE calculator
+        calc = PyACECalculator("potential.yaml")
```

Comment:
Never hardcode the filename here. It should be provided from the input argument. 

change line 357 to `potential_filename: Path | str = 'potential.yaml',`
and update this line to `calc = PyACECalculator(potential_filename)`

This way, it is easy to use a potential file in a different directory by providing it to the `potential_filename`.

Reply:
Done, thanks